aten/src/ATen/native/cuda/LossCTC.cu

```diff
@@ -517,7 +551,9 @@ Tensor ctc_loss_backward_gpu_template(const Tensor& grad_out, const Tensor& log_
   auto input_lengths_t = at::tensor(input_lengths, targets.options().dtype(kLong));
   tg_batch_offsets = tg_batch_offsets.cuda();
 
-  auto log_beta = at::empty_like(log_alpha);
+  Tensor log_beta = at::empty_like(log_alpha);
+  log_beta.fill_(neginf);
```

Comment:
@t-vi  I am not very comfortable to use uninitialized value, even though they are not supposed to be used before assigning any value. There are 3 different GPU kernels to use `log_beta` and each of them has rather complex logic. We have been bit by un-initialized value in the past, so I suggest to initialize them first. 

I  am not sure why I cannot use `at::full_like(log_alpha, neginf)` here -- it generates tons of compiling errors.

Reply:
OK. (`full_like` - strange, especially given that it works for grad).